packages/react/src/lib/accessibility/Behaviors/Menu/menuItemBehavior.ts

```diff
@@ -44,11 +44,18 @@ const menuItemBehavior: Accessibility<MenuItemBehaviorProps> = props => ({
 
   keyActions: {
     root: {
-      performClick: {
-        keyCombinations: [{ keyCode: keyboardKey.Enter }, { keyCode: keyboardKey.Spacebar }],
-      },
+      ...(!props.hasWrapper && {
+        performClick: {
+          keyCombinations: [{ keyCode: keyboardKey.Enter }, { keyCode: keyboardKey.Spacebar }],
+        },
+      }),
     },
     wrapper: {
+      ...(props.hasWrapper && {
+        performClick: {
+          keyCombinations: [{ keyCode: keyboardKey.Enter }, { keyCode: keyboardKey.Spacebar }],
```

Comment:
did you try leaving performClick on root and adding closeAllNenus on wrapper when enter/space is pressed?

Reply:
Yes, but closeAllMenus would be called on the item. Since it contains no submenu, nothing gets closed.

Reply:
let's say we have parent and child menu. You press enter on an item of the child menu. The key handler on the wrapper should get an Enter keydown. That item will not do anything as it does not have a submenu. So the event propagates to the parent menu item, which has a submenu - key handler on the parent warpper will handle it and the closeAllMenus method will close the menu - but maybe my thinking is wrong, can you please debug?

Reply:
I managed to make it work using `closeAllMenus()`. This simplifies the implementation 😇